### PR TITLE
Document MeshDebugPluginMaterial

### PR DESCRIPTION
**Pull request by alexchuber**
Adding a documentation page for using the MeshDebugPluginMaterial. Let me know if this is a good place for it-- I wasn't sure where it should go in the structure.

(Yes, this is very last minute, but such is the beauty of GitHub and open-source. I'll still be available to make changes :))

### CODE REVIEW

File: configuration/structure.json [hunk 1]
```diff
@@ -2513,6 +2513,11 @@
                             "friendlyName": "Display Axes",
                             "children": {},
                             "content": "toolsAndResources/utilities/World_Axes"
+                        },
+                        "VisualizeMeshData": {
```
Comment (Popov72) [inline configuration/structure.json]: I would have put it under Features => Deep Dive => Mesh :)

Utility functions are intended for bits of external code to help do a specific thing. Here, it's more of a debugging option for meshes, that uses code already existing in the library.

Let's see what others think (cc @PirateJC).

**Review by Popov72 — COMMENTED** [review]
LGTM!

File: configuration/structure.json (hunk 1 above)
Comment (RaananW) [reply configuration/structure.json]: I wonder if @alexchuber got the time to do that now. I think we might need to merge it and change it ourselves, but i'll wait a bit to see if she manages to answer this week

Comment (alexchuber) [reply configuration/structure.json]: I'm here! Lol. I figured I was putting it in the wrong place, but I didn't have a good feel for the different sections in the docs.

Comment (alexchuber) [reply configuration/structure.json]: (I'll wait a bit to see if cc'd agree with the new path before committing the change)

**Review by sebavan — APPROVED** [review]

**Pull request merged**